src/main/java/org/graylog2/syslog4j/impl/unix/socket/UnixSocketSyslog.java

```diff
@@ -39,6 +40,12 @@ public void setSunPath(String sunPath) {
             System.arraycopy(sunPath.getBytes(), 0, this.sun_path, 0, sunPath.length());
             System.arraycopy(ZERO_BYTE, 0, this.sun_path, sunPath.length(), 1);
         }
+
+		@Override
+		protected List getFieldOrder() {

```

Comment:
What exactly is this method overriding? Neither `AbstractSyslog` nor `SyslogIF` provide this method.